### PR TITLE
:sparkles: Source Bing-Ads: update docs

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What

- Deleted `docs/integrations/sources/bing-ads.inapp.md`
- Added Prerequisites section
- Changed streams naming to be more user-friendly
- Added Search Query Performance Report stream to reports list
- Added suggestedStreams

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

**Review by davydov-d — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

File: docs/integrations/sources/bing-ads.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
...
 This page contains the setup guide and reference information for the Bing Ads source connector.
 
+## Prerequisites
+- Microsoft Advertising account
+- Microsoft Developer Token
+- Reports start date
+
 ## Setup guide
 
 ### Step 1: Set up Bing Ads
```
Comment (alafanechere) [inline docs/integrations/sources/bing-ads.md:12]: Can we hide this step for Cloud?

**Review by alafanechere — COMMENTED** [review]
Can we differentiate the cloud set up doc and the oss set up doc?
On cloud users are seeing instruction on how to create an oauth app and a refresh token which are not relevant as we have an OAuth implementation on cloud.

File: docs/integrations/sources/bing-ads.md (hunk 1 above)
Comment (darynaishchenko) [reply docs/integrations/sources/bing-ads.md]: added env oss

File: docs/integrations/sources/bing-ads.md [hunk 2]
```diff
@@ -2,22 +2,44 @@
...
+<!-- env:oss -->
+
+For Airbyte Open Source set up your application to get **Client ID**, **Client Secret**, **Refresh Token** 
 
 1. [Register your application](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-register?view=bingads-13) in the Azure portal.
 2. [Request user consent](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-consent?view=bingads-13l) to get the authorization code.
 3. Use the authorization code to [get a refresh token](https://docs.microsoft.com/en-us/advertising/guides/authentication-oauth-get-tokens?view=bingads-13).
 
-:::note
+4. :::note
```
Comment (alafanechere) [inline docs/integrations/sources/bing-ads.md:20]:
```suggestion
:::note
```
The note block is not rendered correctly otherwis
Screenshot 2023-11-03 at 18 39 52
e

File: docs/integrations/sources/bing-ads.md [hunk 3]
```diff
@@ -2,22 +2,44 @@
...
+
+:::info
+
+To use Bing Ads APIs, you must have a developer token and valid user credentials.  You can follow these steps to get a developer token for production.
+    -  Sign in with [Super Admin](https://learn.microsoft.com/en-us/advertising/guides/account-hierarchy-permissions?view=bingads-13#user-roles-permissions) credentials at the [Microsoft Advertising Developer Portal](https://developers.ads.microsoft.com/Account) account tab. 
+    - Choose the user that you want associated with the developer token. Typically an application only needs one universal token regardless how many users will be supported. 
+    - Click on the Request Token button.
+See [Microsoft Advertising docs](https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token) for more info.
+
+:::
```
Comment (alafanechere) [inline docs/integrations/sources/bing-ads.md:40]: Screenshot 2023-11-03 at 18 41 05
Do you know whye steps are rendered inside a code block?

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/bing-ads.md (hunk 2 above)
Comment (darynaishchenko) [reply docs/integrations/sources/bing-ads.md]: fixed

File: docs/integrations/sources/bing-ads.md (hunk 3 above)
Comment (darynaishchenko) [reply docs/integrations/sources/bing-ads.md]: I agree, not the best way to show this steps, I added just list instead of this info block, thanks

**Review by alafanechere — APPROVED** [review]
Thank you @darynaishchenko and congrats for the 32000 PR!

**Pull request merged**